metapackages/auto-instrumentations-node/src/utils.ts

```diff
@@ -134,15 +134,15 @@ export type InstrumentationConfigMap = {
 
 export function getNodeAutoInstrumentations(
   inputConfigs: InstrumentationConfigMap = {}
-): Instrumentation[] {
+): InstrumentationAbstract[] {
```

Comment:
I updated this so I can, in a typesafe manner, rely on the `init()` method being on the instrumentations. An alternative would be to move that method to the `Instrumentation` interface, but that would be a change in `opentelemetry-js` so I wanted to bring it up for discussion before making that change

Reply:
This could also use `InstrumentationBase` all the same